packages/account/src/Components/sent-email-modal/sent-email-modal.tsx

```diff
@@ -75,6 +88,10 @@ const SentEmailModal = ({
 
     const onLiveChatClick = () => {
         onClose();
+
+        // TODO delete this ignore after implementing ts for livechat in core
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore
         window.LiveChatWidget?.call('maximize');
```

Comment:
@yauheni-kryzhyk-deriv this comment above can be removed & this line can be replaced with:
```suggestion
        (
            window as Window & typeof globalThis & { LiveChatWidget: { call: (param: string) => void } }
        ).LiveChatWidget?.call('maximize');
```
🙂

Reply:
thanks, I will implement this :+1: